Author: Jan Dittberner <jandd@debian.org>
Subject: remove bitbucket requirement from sphinx configuration
 The upstream sphinx configuration requires the bitbucket plugin that
 has not yet been packaged for Debian, this patch removes this
 requirement
--- a/docs/sphinx/conf.py
+++ b/docs/sphinx/conf.py
@@ -23,9 +23,9 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = [ 'sphinxcontrib.bitbucket' ]
+#extensions = [ 'sphinxcontrib.bitbucket' ]
 
-bitbucket_project_url = 'http://bitbucket.org/dhellmann/virtualenvwrapper/'
+#bitbucket_project_url = 'http://bitbucket.org/dhellmann/virtualenvwrapper/'
 
 # Add any paths that contain templates here, relative to this directory.
 #templates_path = ['pkg/templates']
